### PR TITLE
Validate Contract Threshold cas-414

### PR DESCRIPTION
**Pull request by tuffon**

### CODE REVIEW

File: backend/main/server/controllers.go [hunk 1]
```diff
@@ -344,6 +351,13 @@ func (a *App) updateCommunity(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	//Validate Contract Thresholds
+	err = validateConractThreshold(*payload.Strategies)
```
Comment (jbluks) [inline backend/main/server/controllers.go:355]: Spelling error - should be validateCon**t**ractThreshold

File: backend/main/server/controllers.go [hunk 2]
```diff
@@ -353,6 +367,17 @@ func (a *App) updateCommunity(w http.ResponseWriter, r *http.Request) {
 	respondWithJSON(w, http.StatusOK, c)
 }
 
+func validateConractThreshold(s []models.Strategy) error {
```
Comment (jbluks) [inline backend/main/server/controllers.go:370]: Could we put this at the bottom of the file with the helpers?

**Review by jbluks — CHANGES_REQUESTED** [review]

**Comment by jbluks** [comment]
Two small changes - but otherwise looks good!

File: backend/main/server/controllers.go (hunk 1 above)
Comment (tuffon) [reply backend/main/server/controllers.go]: nice catch

File: backend/main/server/controllers.go (hunk 2 above)
Comment (tuffon) [reply backend/main/server/controllers.go]: sure thing!

**Review by jbluks — APPROVED** [review]

**Pull request merged**